project/build.properties

```diff
@@ -15,4 +15,4 @@
  * limitations under the License.
  */
 
-sbt.version=1.2.7
+sbt.version=1.3.9
```

Comment:
We also reference the sbt version in ``containers/release-candidate/setup-container.sh``

Reply:
ok